website/static/css/custom.css

```diff
@@ -0,0 +1,18 @@
+.fixedHeaderContainer header img {
+    height: 80%;
+}
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
```

Comment:
Do we need these?

Reply:
@kevinbarabash This file was provided by `docusaurus` as a boilerplate. I think we can remove them, if we don't have any custom styles to apply.